README.md

```diff
@@ -49,6 +49,26 @@ module "kafka_instance" {
 }
 ```
 
+### Example with additional EC2 servers and volumes and EIP
+
+```terraform
+module "kafka_instance" {
+  source                      = "git::https://github.com/cloudposse/terraform-aws-ec2-instance.git?ref=master"
+  ssh_key_pair                = "${var.ssh_key_pair}"
+  vpc_id                      = "${var.vpc_id}"
+  security_groups             = ["${var.security_groups}"]
+  subnet                      = "${var.subnet}"
+  associate_public_ip_address = "true"
+  name                        = "kafka"
```

Comment:
I think we should make `namespace`, `stage` and `name` consistently the first parameters below `source` across all examples

Reply:
will update